R/as_phylo.R

```diff
@@ -0,0 +1,42 @@
+#' @rdname as_phylo
+#' @title as_phylo
+#' @description `r lifecycle::badge("deprecated")`
+#'
+#' `as_phylo` methods were renamed `as.phylo`
+#' for more consistency with other R methods.
+#'
+#' @family community
+#' @export
+#' @param x object to be coerced
+#' @inheritParams ape::as.phylo
+as_phylo <- function(x, ...) {
+  lifecycle::deprecate_soft(
+    "1.5.0",
+    "ape::as.phylo()"
+  )
+  UseMethod("ape::as.phylo")
```

Comment:
I'm not sure that this is going to work. This should check that ape is installed and forward to `ape::as.phylo()`, with a deprecation message. This is no longer a generic, we don't need `UseMethod()` here.

Or, alternatively, with or without deprecation message: